server/services/lostBedService.test.ts

```diff
@@ -47,11 +49,26 @@ describe('LostBedService', () => {
       const token = 'SOME_TOKEN'
       lostBedClient.find.mockResolvedValue(lostBed)
 
-      const postedLostBed = await service.getLostBed(token, 'premisesID', 'id')
+      const postedLostBed = await service.getLostBed(token, premisesId, 'id')
 
       expect(postedLostBed).toEqual(lostBed)
       expect(LostBedClientFactory).toHaveBeenCalledWith(token)
-      expect(lostBedClient.find).toHaveBeenCalledWith('premisesID', 'id')
+      expect(lostBedClient.find).toHaveBeenCalledWith(premisesId, 'id')
+    })
+  })
+
+  describe('getLostBeds', () => {
+    it('on success returns the lostBeds for a premises', async () => {
+      const expectedLostBeds: Array<LostBed> = lostBedFactory.buildList(2)
```

Comment:
No need to change this as it's not a big deal but I don't think you have to type `expectedLostBeds` here